Packs/CortexXDR/Playbooks/Cortex_XDR_-_quarantine_file.yml

```
@@ -0,0 +1,222 @@
+id: Cortex_XDR_-_quarantine_file
+version: -1
+name: Cortex_XDR_-_quarantine_file
```

playbook name should be readable and pretty formatted (update the id as well)
```suggestion
name: Cortex XDR - Quarantine File
```